app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,86 @@
+class CustomersController < ApplicationController
+  rescue_from ActiveRecord::RecordNotFound, with: :catch_not_found
+  before_action :set_customer, only: %i[ show edit update destroy ]
+
+  # GET /customers or /customers.json
+  def index
+    @customers = Customer.all
+  end
+
+  # GET /customers/1 or /customers/1.json
+  def show
+  end
+
+  # GET /customers/new
+  def new
+    @customer = Customer.new
+  end
+
+  # GET /customers/1/edit
+  def edit
+  end
+
+  # POST /customers or /customers.json
+  def create
+  #   @customer = Customer.new(customer_params)
+
+  #   respond_to do |format|
+  #     if @customer.save
+  #       format.html { redirect_to customer_url(@customer), notice: "Customer was successfully created." }
+  #       format.json { render :show, status: :created, location: @customer }
+  #     else
+  #       format.html { render :new, status: :unprocessable_entity }
+  #       format.json { render json: @customer.errors, status: :unprocessable_entity }
+  #     end
+  #   end
+    @customer = Customer.new(customer_params)
```

Comment:
Careful! You'll need branching logic here similar to in the update action to catch any errors. Right now you can try to create an invalid customer and it will say that it was created successfully.